### ISSUE TITLE
Could you provide a script for downloading the training dataset?

### ISSUE THREAD

**Issue by hiyyg** [NVlabs/FoundationPose#201]

**Comment by wenbowen123** [comment]
you can consider using [gdown](https://github.com/wkentaro/gdown) to auto download.

**Comment by hiyyg** [comment]
Thanks. Could you provide a file list for each folder of the train_dataset?

**Comment by W-QY** [comment]
> Thanks. Could you provide a file list for each folder of the train_dataset?

Hello, have you completed the download and processing of the foundationpose training dataset? I hope to discuss it with you.

**Comment by hiyyg** [comment]
> > Thanks. Could you provide a file list for each folder of the train_dataset?
> 
> Hello, have you completed the download and processing of the foundationpose training dataset? I hope to discuss it with you.

Not yet. Have you completed it?

**Comment by W-QY** [comment]
> > > Thanks. Could you provide a file list for each folder of the train_dataset?
> > 
> > 
> > Hello, have you completed the download and processing of the foundationpose training dataset? I hope to discuss it with you.
> 
> Not yet. Have you completed it?

Hello, I recently downloaded the objaverse mesh file and some datasets used in the paper. Now I am trying to convert the pose labels in the dataset into pose labels in BOP format. Are you doing something similar?

**Comment by hiyyg** [comment]
> > > > Thanks. Could you provide a file list for each folder of the train_dataset?
> > > 
> > > 
> > > Hello, have you completed the download and processing of the foundationpose training dataset? I hope to discuss it with you.
> > 
> > 
> > Not yet. Have you completed it?
> 
> Hello, I recently downloaded the objaverse mesh file and some datasets used in the paper. Now I am trying to convert the pose labels in the dataset into pose labels in BOP format. Are you doing something similar?

Hi @W-QY , I am trying a similar thing, but I have not figured out how to parse the camera intrinsic and the object pose from the downloaded data. Have you figured it out?

**Comment by W-QY** [comment]
> > > > > Thanks. Could you provide a file list for each folder of the train_dataset?
> > > > 
> > > > 
> > > > Hello, have you completed the download and processing of the foundationpose training dataset? I hope to discuss it with you.
> > > 
> > > 
> > > Not yet. Have you completed it?
> > 
> > 
> > Hello, I recently downloaded the objaverse mesh file and some datasets used in the paper. Now I am trying to convert the pose labels in the dataset into pose labels in BOP format. Are you doing something similar?
> 
> Hi @W-QY , I am trying a similar thing, but I have not figured out how to parse the camera intrinsic and the object pose from the downloaded data. Have you figured it out?

Hi，You can refer to this[https://github.com/NVlabs/FoundationPose/issues/217](url)
